VisualStudio.gitignore

```diff
@@ -263,3 +263,6 @@ paket-files/
 # Python Tools for Visual Studio (PTVS)
 __pycache__/
 *.pyc
+
+# Cake
+tools/
```

Comment:
I can't help but feel this is too broad a rule, and might catch other things.

For example, FAKE uses a `.fake/` path for it's local cache.